Loop/Managers/RemoteDataServicesManager.swift

```diff
@@ -6,165 +6,387 @@
 //  Copyright © 2016 Nathan Racklyeft. All rights reserved.
 //
 
+import os.log
 import Foundation
 import LoopKit
 
-final class RemoteDataServicesManager: CarbStoreSyncDelegate {
+protocol RemoteDataServicesManagerDelegate: AnyObject {
```

Comment:
As discussed before, a delegate protocol for providing dependencies is unneeded indirection at this level. Created https://tidepool.atlassian.net/browse/LOOP-969 to track straightening this out.

Reply:
Thanks!